internal/pkg/manifest/pipeline.go

```diff
@@ -178,8 +183,9 @@ type Source struct {
 
 // Build defines the build project to build and test image.
 type Build struct {
-	Image     string `yaml:"image"`
-	Buildspec string `yaml:"buildspec,omitempty"`
+	Image            string            `yaml:"image"`
+	Buildspec        string            `yaml:"buildspec,omitempty"`
+	AdditionalPolicy *AdditionalPolicy `yaml:"additional_policy,omitempty"`
```

Comment:
what would you think about just having this as `yaml.Node` or `map[string]any` and marshaling it directly into the pipeline template? and just letting cloudformation do the validation?

Reply:
The only thing I thought of is CFN template errors are little difficult for the customers to understand and solve that is why I kept it this way that we will guide customers with more user friendly errors.

Reply:
Yeah but manifest validation cannot guarantee their input is entirely correct. Most likely those users who need to specify additional policies they already have the json policy document handy.